nix/ucm.nix

```diff
@@ -97,13 +97,35 @@ in
         --prefix LD_LIBRARY_PATH : ${lib.makeLibraryPath [ libb2 openssl curl ]} \
         --add-flags "--runtime-path $out/lib/runtime/bin/unison-runtime" \
         --set-default UCM_WEB_UI "$out/ui"
+
+      makeWrapper $out/unison/unison $out/unison/unison-no-runtime \
+        --prefix PATH : ${binPath} \
+        --prefix LD_LIBRARY_PATH : ${lib.makeLibraryPath [ libb2 openssl curl ]} \
     '';
 
     postFixup = ''
-      installShellCompletion --cmd ucm \
-        --bash <(${ucm} --bash-completion-script ${ucm}) \
-        --fish <(${ucm} --fish-completion-script ${ucm}) \
-        --zsh <(${ucm} --zsh-completion-script ${ucm})
+      $out/unison/unison-no-runtime --bash-completion-script $out/unison/unison-no-runtime
```

Comment:
It can't run `$out/unison/unison` at all. If you run `ldd` on it, you'll get something like

```
        
ldd $out/unison/unison
        linux-vdso.so.1 (0x00007ffff7fc5000)
        libm.so.6 => /nix/store/3dyw8dzj9ab4m8hv5dpyx7zii8d0w6fi-glibc-2.39-52/lib/libm.so.6 (0x00007ffff7ed9000)
        libpthread.so.0 => /nix/store/3dyw8dzj9ab4m8hv5dpyx7zii8d0w6fi-glibc-2.39-52/lib/libpthread.so.0 (0x00007ffff7ed4000)
        libz.so.1 => not found
        librt.so.1 => /nix/store/3dyw8dzj9ab4m8hv5dpyx7zii8d0w6fi-glibc-2.39-52/lib/librt.so.1 (0x00007ffff7ecf000)
        libutil.so.1 => /nix/store/3dyw8dzj9ab4m8hv5dpyx7zii8d0w6fi-glibc-2.39-52/lib/libutil.so.1 (0x00007ffff7eca000)
        libdl.so.2 => /nix/store/3dyw8dzj9ab4m8hv5dpyx7zii8d0w6fi-glibc-2.39-52/lib/libdl.so.2 (0x00007ffff7ec3000)
        libgmp.so.10 => not found
        libc.so.6 => /nix/store/3dyw8dzj9ab4m8hv5dpyx7zii8d0w6fi-glibc-2.39-52/lib/libc.so.6 (0x00007ffff7ccc000)
        /lib64/ld-linux-x86-64.so.2 => /nix/store/3dyw8dzj9ab4m8hv5dpyx7zii8d0w6fi-glibc-2.39-52/lib64/ld-linux-x86-64.so.2 (0x00007ffff7fc7000)
```